ext/intl/calendar/calendar_methods.cpp

```diff
@@ -1085,6 +1079,7 @@ U_CFUNC PHP_FUNCTION(intlcal_to_date_time)
 	object_init_ex(return_value, php_date_get_date_ce());
 	zend_call_known_instance_method_with_2_params(
 		Z_OBJCE_P(return_value)->constructor, Z_OBJ_P(return_value), NULL, &ts_zval, timezone_zval);
+	// TODO Bubble up exception?
```

Comment:
Actually before I merge it, @nikic should I do that in this PR or just leave the TODO comment?

Reply:
Looks ok to change.